http/rest-client-reactive/src/main/java/io/quarkus/ts/http/restclient/reactive/files/FileResource.java

```diff
@@ -30,14 +30,14 @@ public class FileResource {
     private final OsUtils utils;
     private final List<File> deathRow = new LinkedList<>();
 
-    public FileResource(@ConfigProperty(name = "client.filepath") Optional<String> folder) {
+    public FileResource(@ConfigProperty(name = "client.filepath", defaultValue = "/tmp") Optional<String> folder) {
```

Comment:
Is this Windows-friendly?

Reply:
Ah, I already accepted it :-(, shouldn't CI Windows run ensure it is?

Reply:
Truth is I missed this one, very sorry.

Reply:
Tests that depend on the file ^^ are disabled on Windows or they are disabled due to Quarkus issues, that's why it didn't fail.